bin/p2-exec/main.go

```diff
@@ -29,6 +32,8 @@ var (
 	nolim          = kingpin.Flag("nolimit", "Remove rlimits.").Short('n').Bool()
 	clearEnv       = kingpin.Flag("clearenv", "Clear all environment variables before loading envDir(s).").Bool()
 	workDir        = kingpin.Flag("workdir", "Set working directory.").Short('w').String()
+	umask          = kingpin.Flag("umask", "Set the process umask. Use octal notation ex. 0022").Short('m').Default(umaskSentinal).String()
+	umaskSentinal  = ""
```

Comment:
I was too subtle in my last comment: "Sentinal" is misspelled. Use "Sentinel" (or don't name this constant).